codes/models/modules/architectures/DVDNet_arch.py

```diff
@@ -0,0 +1,68 @@
+import torch
+import torch.nn as nn
+
+def vertcal_upscale(x, upfield=True):
+
+    n, c, h, w = x.shape
+    h *= 2

```

Comment:
Is there any possibility of h not being exactly twice the size?

Reply:
Because real interlaced data is made of fields that are both the same size, there's no way for adding both of those together to not equal exactly h*2. I suppose someone could input something that isn't like real world interlacing but tbh I don't really see that being an issue

Reply:
Yeah both fields HAVE to be the same height due to how it works, so this isn't a problem.